usr/src/uts/common/fs/zfs/sys/uberblock_impl.h

```diff
@@ -54,6 +54,9 @@ struct uberblock {
 
 	/* highest SPA_VERSION supported by software that wrote this txg */
 	uint64_t	ub_software_version;
+
+	/* reserved */
+	uint64_t	padding[3];
```

Comment:
This is OK, but I think it would be more clear to spell out the fields as they will be in the final state, with comments explaining that they are not in use yet:
```
/* These fields are reserved for features that are under development: */
uint64_t ub_mmp_magic;
uint64_t ub_mmp_delay;
uint64_t ub_checkpoint_txg;
```